From bfffd2c851b2b8ed2357cb89c93539c21112d638 Mon Sep 17 00:00:00 2001
From: Ahmed Ismail <Ahmed.Ismail@arm.com>
Date: Tue, 27 Jan 2026 17:01:00 +0000
Subject: [PATCH] ethos-u-core-driver: Fix invalid URL

The review.mlplatform url is not valid anymore. Hence,
switch to using the new git.gitlab.arm domain.

Signed-off-by: Ahmed Ismail <Ahmed.Ismail@arm.com>
---
 lib/ext/ethos_u_core_driver/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/ext/ethos_u_core_driver/CMakeLists.txt b/lib/ext/ethos_u_core_driver/CMakeLists.txt
index 18e277751..fb02230dc 100644
--- a/lib/ext/ethos_u_core_driver/CMakeLists.txt
+++ b/lib/ext/ethos_u_core_driver/CMakeLists.txt
@@ -10,7 +10,7 @@ fetch_remote_library(
     LIB_SOURCE_PATH_VAR     ETHOS_DRIVER_PATH
     LIB_PATCH_DIR           ${CMAKE_CURRENT_LIST_DIR}
     FETCH_CONTENT_ARGS
-        GIT_REPOSITORY      "https://review.mlplatform.org/ml/ethos-u/ethos-u-core-driver"
+        GIT_REPOSITORY      "https://git.gitlab.arm.com/artificial-intelligence/ethos-u/ethos-u-core-driver.git"
         GIT_TAG             "24.05"
         GIT_SHALLOW         TRUE
         GIT_PROGRESS        TRUE
-- 
2.34.1

